ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -507,14 +508,16 @@ static void maybeCreateProcessingLogStream(
 
     final PreparedStatement<?> statement = ProcessingLogServerUtils
         .processingLogStreamCreateStatement(config, ksqlConfig);
+    final Supplier<ConfiguredStatement<?>> configured = () -> ConfiguredStatement.of(
+        statement, Collections.emptyMap(), ksqlConfig);
```

Comment:
Can a ConfiguredStatement be initialized once here instead of using a Supplier? If I understand, the ConfiguredStatement just wraps 3 parameters into one class, and then passes it to the execute() and enqueueCommand() methods. Can one single initialization be enough?